src/Utils/Floats.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Utils;
+
+use Nette;
+
+
+/**
+ * Floats compare tools library.
+ */
+class Floats
+{
+	use Nette\StaticClass;
+
+	/** @var float */
+	private const EPSILON = 1e-10;
+
+
+	public static function isZero(float $value): bool
+	{
+		return abs($value) < self::EPSILON;
+	}
+
+
+	public static function isInteger(float $value): bool
```

Comment:
Epsilon should be user-defined. :)

Reply:
Now it is public static property :)

Reply:
Changed back to constant. If it turns out why it should be user-defined, then we make an adjustment.